packages/model-viewer/src/three-components/gltf-instance/ModelViewerGLTFInstance.ts

```diff
@@ -185,21 +178,38 @@ export class ModelViewerGLTFInstance extends GLTFInstance {
       clone.normalMap = material.normalMap.clone();
       clone.normalMap.needsUpdate = true;
     }
-    if (material.aoMap != null) {
-      clone.aoMap = material.aoMap.clone();
-      clone.aoMap.needsUpdate = true;
-    }
     if (material.emissiveMap != null) {
       clone.emissiveMap = material.emissiveMap.clone();
       clone.emissiveMap.needsUpdate = true;
     }
-    if (material.metalnessMap != null) {
+
+    const roughnessMap: Texture|null = material!.roughnessMap!.clone();
+
+    if (roughnessMap !== null) {
+      roughnessMap.needsUpdate = true;
+      clone.roughnessMap = roughnessMap;
+
+      const {threeRenderer, roughnessMipmapper} = Renderer.singleton;
+      // XR must be disabled while doing offscreen rendering or it will
+      // clobber the camera.
+      const {enabled} = threeRenderer.xr;
+      threeRenderer.xr.enabled = false;
+      roughnessMipmapper.generateMipmaps(clone as MeshStandardMaterial);
```

Comment:
Okay, so this looks like what I missed in my first version. This shouldn't be necessary, so we should try to figure out why this is broken in three. Still, good enough for now.

Reply:
Agreed, I will setup a test case to dig further, if nothing else it will provide a repro case to give to threejs (assuming it does repro)